dist/react-filterbar.js

```diff
@@ -11882,6 +11882,11 @@ var FilterBarStore = exports.FilterBarStore = (function () {
         return this.getConfigurationUrl() !== undefined;
       }
     },
+    isExportable: {
+      value: function isExportable() {
+        return this.getExportResultsUrl() !== undefined;
```

Comment:
I'm assuming this means a "user with no authority to export data" will have an undefined export results URL.